src/main/java/service/operations/BalanceOperationImpl.java

```diff
@@ -0,0 +1,12 @@
+package service.operations;
+
+import db.Storage;
+import model.FruitTransaction;
+import service.OperationHandle;
+
+public class BalanceOperationImpl implements OperationHandle {
```

Comment:
Same logic for other classes.
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```